lib/astUtils.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * @fileoverview Common utils for AST.
+ * @author Gyandeep Singh
+ */
```

Comment:
Add copyright and the line separators for each section.

Also, filename should be ast-utils.js.